src/Microsoft.AspNetCore.Localization/RequestLocalizationOptions.cs

```diff
@@ -109,5 +109,53 @@ public RequestCulture DefaultRequestCulture
         /// </list>
         /// </summary>
         public IList<IRequestCultureProvider> RequestCultureProviders { get; set; }
+
+        
+        /// <summary>
+        /// Adds the set of the supported cultures by the application.
+        /// </summary>
+        /// <param name="cultures">The cultures to be added.</param>
+        /// <returns>The <see cref="RequestLocalizationOptions"/>.</returns>
+        public RequestLocalizationOptions AddSupportedCultures(params string[] cultures)
+        {
+            var supportedCultures = new List<CultureInfo>();
+
+            foreach (var culture in cultures)
+            {
+                supportedCultures.Add(new CultureInfo(culture));
+            }
+
+            SupportedCultures = supportedCultures;
+            return this;
+        }
+
+        /// <summary>
+        /// Adds the set of the supported UI cultures by the application.
+        /// </summary>
+        /// <param name="uiCultures">The UI cultures to be added.</param>
+        /// <returns>The <see cref="RequestLocalizationOptions"/>.</returns>
+        public RequestLocalizationOptions AddSupportedUICultures(params string[] uiCultures)
```

Comment:
Should we have overloads that also accept `CultureInfo` ?

Reply:
I'm not sure why we need another overload while we already have `SupportedUICultures` property!!

Reply:
Ok